### PR TITLE
Small README update

### PR DESCRIPTION
**Pull request by kvark**

### CODE REVIEW

**Comment by kvark** [comment]
bors r=msiglreith

**Comment by notriddle** [comment]
https://www.traviscistatus.com/incidents/p3wv8zvkz3zb

File: README.md [hunk 1]
```diff
@@ -71,9 +77,8 @@ Biggest open-source projects are:
   - [ggez](https://github.com/ggez/ggez) engine
   - Piston engine - [2d graphics](https://github.com/PistonDevelopers/gfx_graphics)
   - [LazyBox](https://github.com/lazybox/lazybox) engine
-  - [Zone of Control](https://github.com/ozkriff/zoc) game
```
Comment (ozkriff) [inline README.md]: whyy? :'-(

**Comment by kvark** [comment]
Because it's abandoned

**Comment by kvark** [comment]
bors r=msiglreith

**Comment by notriddle** [comment]
https://travis-ci.org/gfx-rs/gfx/jobs/262678359

**Pull request merged**